Makefile

```diff
@@ -477,7 +477,6 @@ _test-unit-python: _build-release-image-python
 	--workdir=/usr/local/lib/python3.8/site-packages/ostk/physics \
 	--entrypoint="" \
 	--volume="$(project_directory)/share:/app/share" \
-	--env=OSTK_PHYSICS_COORDINATE_FRAME_PROVIDERS_IERS_MANAGER_LOCAL_REPOSITORY \
```

Comment:
Should we keep this, and modify hardcoded URLs in unit tests instead w/ `os.environ.get('OSTK_PHYSICS_COORDINATE_FRAME_PROVIDERS_IERS_MANAGER_LOCAL_REPOSITORY')`?

Reply:
Adopted that solution